website/js/index.js

```diff
@@ -448,6 +479,84 @@ var VariantDetail = React.createClass({
 				</tr>);
         });
 
+        var versionRows = [];
+        // which keys represent comma seperated lists? for filtering out re-ordered lists
+        var listKeys = [
+            "Submitter_ClinVar",
+            "Method_ClinVar",
+            "Source",
+            "Date_Last_Updated_ClinVar",
+            "Source_URL",
+            "SCV_ClinVar",
+            "Clinical_Significance_ClinVar",
+            "Allele_Origin_ClinVar"
+        ];
+        for (var i = 0; i < data.length; i++) {
+            let version = data[i],
+                changes = [],
+                release = version["Data_Release"], //eslint-disable-line dot-notation
+                hightlightRow = false;
+
+            // if this is not the oldest version of the variant, diff them
+            if (i < data.length - 1) {
+                let previous = data[i + 1];
+                if (version["Pathogenicity_expert"] !== previous["Pathogenicity_expert"]) { //eslint-disable-line dot-notation
+                    hightlightRow = true;
+                }
+                for (var key in version) {
+                    if (!_.contains(["Data_Release", "Change_Type", "id"], key) && version[key] !== previous[key]) { //eslint-disable-line dot-notation
+                        // handle nested list in Pathogenicity_all
+                        if (key === "Pathogenicity_all") {
+                            let pathogenicityCurrent = _.map(version[key].split(';'), elem => elem.split(',').sort().join(','));
+                            let pathogenicityPrevious = _.map(previous[key].split(';'), elem => elem.split(',').sort().join(','));
+                            let added = _.map(_.difference(pathogenicityCurrent, pathogenicityPrevious), elem => `+${elem.trim().replace(/_/g, " ")}`);
+                            let deleted = _.map(_.difference(pathogenicityPrevious, pathogenicityCurrent), elem => `-${elem.trim().replace(/_/g, " ")}`);
+                            if (added.length || deleted.length) {
+                                changes.push(
+                                    <span>
+                                        <strong>{ getDisplayName(key) }: </strong> <br />
+                                        { added.join(', ') }{ !!(added.length && deleted.length) && ', '}{ deleted.join(', ') }
+                                    </span>, <br />
+                                );
+                            }
+                        }
+                        else if (_.contains(listKeys, key)) {
```

Comment:
The case for pathogenicity_all is pretty weird. This could definitely use some refactoring, I think it might be best to move the variant history table into it's own component even.